fakeredis/aioredis.py

```diff
@@ -1,9 +1,8 @@
-import distutils.version
-
 import aioredis
+import packaging.version
```

Comment:
I would probably use the same scheme as all the other files (`from packaging.version import Version`), but that's a very minor suggestion.

Reply:
I wanted to avoid exposing a `fakeredis.aioredis.Version`. I know I could import it as `_Version` or something but that still wouldn't be consistent.

Reply:
It's a public module. `fakeredis._aioredis1` and `fakeredis._aioredis2` are internal.